charts/airflow/values.yaml

```diff
@@ -1506,6 +1506,9 @@ externalRedis:
   ## the key within `externalRedis.passwordSecret` containing the password string
   ##
   passwordSecretKey: "redis-password"
+  ## the connection properties for external reids. for eg ?ssl_cert_reqs=CERT_OPTIONAL
```

Comment:
Please add a newline between the `passwordSecretKey` value and the docstring of `properties`